patches/0011-Replace-RtlGetNtVersionNumbers-with-RtlGetVersion.patch

```diff
@@ -0,0 +1,263 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: qmuntal <quimmuntal@gmail.com>
+Date: Tue, 12 Mar 2024 14:20:33 +0100
+Subject: [PATCH] Replace RtlGetNtVersionNumbers with RtlGetVersion
+
+This is a cherry-pick of CL 571015 (commit cff7267e0d77f02d582c613c272b6f8ebf1c0412),
+as upstream won't backport it to Go 1.22.
+
+The cherry-pick is not completely clean, it needed some manual conflict
+resolution.
```

Comment:
Maybe mention 565595: net: consolidate the existing Windows version checks | https://go-review.googlesource.com/c/go/+/565595 ?

Mentioning which files required manual resolution would also be nice (generally the generated Git commit desc will include a list of what had conflicts). Although not necessary if it's not easy to get now IMO.